docs/userguide/modules/creating-service-bindings/pages/binding-options.adoc

```diff
@@ -236,5 +239,12 @@ spec:
 If a service binding is declared using these label selectors to pick up workloads, SBO will
 periodically attempt to find and bind new workloads that match the given label selector.
```

Comment:
This sentence was not there before. Have you added it newly now?

This is already written from L189 to L191 in the 1st para. No need to duplicate it here. Suggest to remove this sentence from here.

Reply:
Nope, it's in latest `master`.  I haven't added this recently; it was added when the label selector documentation PR merged.

Reply:
Taken out, thanks.

Reply:
@sadlerap I can see only this change and not the others